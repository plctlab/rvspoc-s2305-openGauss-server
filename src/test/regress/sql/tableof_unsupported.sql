------ Prepare ------
-- create type of TYPTYPE_TABLEOF
create type r0 is (c1 int1, c2 int2);

create type t0 is table of int4;
create type t1 is (c1 int1, c2 t0);
create type t2 is table of r0;

-- create table
create table tableof_unsupported(id t0);
create table tableof_unsupported(id t1);
create table tableof_unsupported(id t2);