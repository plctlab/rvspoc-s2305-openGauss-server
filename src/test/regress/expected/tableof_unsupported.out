------ Prepare ------
-- create type of TYPTYPE_TABLEOF
create type r0 is (c1 int1, c2 int2);
create type t0 is table of int4;
create type t1 is (c1 int1, c2 t0);
create type t2 is table of r0;
-- create table
create table tableof_unsupported(id t0);
ERROR:  type "t0" is not supported as column type
DETAIL:  "t0" is a nest table type
create table tableof_unsupported(id t1);
ERROR:  type "t0" is not supported as column type
DETAIL:  "t0" is a nest table type
create table tableof_unsupported(id t2);
ERROR:  type "t2" is not supported as column type
DETAIL:  "t2" is a nest table type
